clients/python/README.md

```diff
@@ -24,13 +24,17 @@ $ python3 setup.py install
 
 ## Usage
 
+### Metadata Collection
```

Comment:
Similar feedback as above in the `marquez-java` usage: https://github.com/MarquezProject/marquez/pull/1762/files#r748589744